build/test/OverlaySdkOnLKG.targets

```diff
@@ -11,4 +12,8 @@
     <Copy SourceFiles="@(OverlayCLI)"
           DestinationFiles="@(OverlayCLI->'$(SdkOutputDirectory)\sdk\$(DotNetCoreSdkLKGVersion)\%(RecursiveDir)%(Filename)%(Extension)')" />
   </Target>
+
+  <Target Name="PublishRedist">
+    <Exec Command="$(DotNetExe) publish $(RepoRoot)/src/redist/redist.csproj --no-build --output $(CliOutputDirectory)" WorkingDirectory="$(RepoRoot)/src/redist" />
```

Comment:
Is there a reason to shell out to a new process for this?  That has the same downside as the `DotNetPublish` target would have (ie no logging).

Reply:
This was already moved to msbuild in a subsequent commit.